lib/Minilla/ModuleMaker/ModuleBuild.pm

```diff
@@ -75,6 +75,9 @@ use strict;
 use warnings;
 use utf8;
 
+? if ( $project->build_class ne 'Module::Build' ) {
```

Comment:
if minil.toml is
```
name = "Foo"
module_maker = "ModuleBuild"

[XSUtil]
needs_compiler_c99 = 1
```
then build_class is Module::Build::XSUtil. I think we can exclude 'Module::Build::XSUtil' too.